### ISSUE TITLE
Map in the meshbuilder

### ISSUE THREAD

**Issue by jroeste** [4DModeller/fdmr#243]
**Describe the bug**
Map is outside its boundaries, overlapping with the mesh builder tools/ interface. 
![image](https://github.com/4DModeller/fdmr/assets/25384408/0e6ada46-cd0d-4e82-ac8f-a33297f31f9e)

**Expected behavior**
Should not overlap with the interface.

**To Reproduce**
Tried to open and close the markdown. 

**User Environment (please complete the following information):**

Gaussian tutorial,

 Windows:
 - R-version 4.3.1,
 - RStudio-version: RStudio 2023.06.0+421 "Mountain Hydrangea" 

>  - SessionInfo(): R version 4.3.1 (2023-06-16 ucrt)
> Platform: x86_64-w64-mingw32/x64 (64-bit)
> Running under: Windows 10 x64 (build 19045)
> 
> Matrix products: default
> 
> 
> locale:
> [1] C
> system code page: 65001
> 
> time zone: Europe/Oslo
> tzcode source: internal
> 
> attached base packages:
> [1] stats     graphics  grDevices utils     datasets  methods   base     
> 
> other attached packages:
> [1] shiny_1.7.5.1   leaflet_2.2.0   INLA_23.10.19-1 sp_2.1-1        Matrix_1.6-1.1  fmesher_0.1.3  
> 
> loaded via a namespace (and not attached):
>  [1] bslib_0.5.1             xfun_0.40               htmlwidgets_1.6.2       lattice_0.21-8         
>  [5] leaflet.providers_2.0.0 vctrs_0.6.4             tools_4.3.1             crosstalk_1.2.0        
>  [9] generics_0.1.3          curl_5.1.0              parallel_4.3.1          tibble_3.2.1           
> [13] proxy_0.4-27            fansi_1.0.5             pkgconfig_2.0.3         KernSmooth_2.23-21     
> [17] RColorBrewer_1.1-3      lifecycle_1.0.3         stringr_1.5.0           compiler_4.3.1         
> [21] farver_2.1.1            MatrixModels_0.5-2      munsell_0.5.0           httpuv_1.6.11          
> [25] sass_0.4.7              htmltools_0.5.6.1       class_7.3-22            yaml_2.3.7             
> [29] pillar_1.9.0            later_1.3.1             jquerylib_0.1.4         ellipsis_0.3.2         
> [33] classInt_0.4-10         cachem_1.0.8            mime_0.12               inlabru_2.9.0          
> [37] tidyselect_1.2.0        digest_0.6.33           stringi_1.7.12          shinybusy_0.3.1        
> [41] sf_1.0-14               dplyr_1.1.3             fdmr_0.1.0              splines_4.3.1          
> [45] fastmap_1.1.1           grid_4.3.1              colorspace_2.1-0        cli_3.6.1              
> [49] magrittr_2.0.3          utf8_1.2.3              e1071_1.7-13            withr_2.5.1            
> [53] scales_1.2.1            promises_1.2.1          lubridate_1.9.3         timechange_0.2.0       
> [57] rmarkdown_2.25          memoise_2.0.1           evaluate_0.22           knitr_1.44             
> [61] rlang_1.1.1             Rcpp_1.0.11             xtable_1.8-4            glue_1.6.2             
> [65] DBI_1.1.3               rstudioapi_0.14         jsonlite_1.8.7          R6_2.5.1               
> [69] plyr_1.8.9              fs_1.6.3                units_0.8-4            
> 

 - OS: [e.g. macOS, Windows, Ubuntu]
 - Version of R
 - Version of fdmr you are using

**Comment by KateHeerema** [comment]
It also doesn't render the map properly in safari/chrome

**Comment by XueqingYin** [comment]
Thanks for creating this issue @jroeste @KateHeerema. It appears that there may be some formatting problems with our geographical data for Bristol (i.e., spatial_dataBris.rds),  so the app cannot render the map properly based on that. I will investigate this and try to find a solution.

**Comment by XueqingYin** [comment]
I haven't figured out the reason, but the app seems to be able to render the map properly after I opened a new R session to run  the code. Below is a normal map I got:

![image](https://github.com/4DModeller/fdmr/assets/83759494/21e24622-1288-41bf-8ce5-3025af05ccf0)

**Comment by jroeste** [comment]
Hey again! 
Actually this happens with another dataset as well:
![image](https://github.com/4DModeller/fdmr/assets/25384408/7b74781b-446e-4ce0-8b35-880063ab24e3)

[snow_df_uio.csv](https://github.com/4DModeller/fdmr/files/13249073/snow_df_uio.csv)

**Comment by mnky9800n** [comment]
is this a bootstrap problem?

**Comment by jroeste** [comment]
@mnky9800n , the csv-file attached above contains the 96 locations for the observations (LON/LAT), including only the values of snow depth. In the netCDF-file, there are several other variables, as defined in the text below (pasted in from the introduction to the tutorial) 

_This tutorial explores whether 4Dmodeller can be used to model the spatial distribution of snow in the Bayelva area on Svalbard, based on once-a-year point-based manual snow depth measurements for several years and snow cover duration derived from time lapse camera imagery (Aalstad et al., 2020). This data has been used before for snow data assimilation (Aalstad et al., 2018) and permafrost modeling (Zweigel et al., 2021) using modeling and data assimilation methods (see references). Additionally, we included elevation and other derived topographic parameters (slope, aspect, topographic position index) as these are known to influence the snow depth distribution. These topographic parameters were derived from a high resolution digital elevation model (Boike et al., 2019). The dataset used in this tutorial has been prepared by Kristoffer Aalstad as a netCDF by combining and processing the following publicly available datasets that have been used in several studies (see literature references):
Westermann, S. (2020).Snow depth and ground surface temperature at Bayelva research site, Svalbard [Data set]. Norstore. https://doi.org/10.11582/2020.00024
Aalstad, K., and Westermann. S. (2021). ACS_Bayelva_class: 302 high-resolution snow cover maps covering the 2012-2017 snowmelt seasons in the Bayelva catchment (Svalbard, Norway) (2.0) [Data set]. Zenodo. https://doi.org/10.5281/zenodo.5010944
Boike, J., Juszak, I., Lange, S., Chadburn, S., Burke, E., Overduin, P. P., Roth, K., Ippisch, O., Bornemann, N., Stern, L., Gouttevin, I., Hauber, E., & Westermann, S. (2018). HRSC-AX data products (DEM and multi channel) from aerial overflights in 2008 over Bayelva (Brøggerhalvøya peninsula, Spitsbergen). (Version 1) [Data set]. Zenodo. https://doi.org/10.1594/PANGAEA.884730
The dataset is originally projected (UTM zone 33X) and available in the data sub-folder in this repository (data/Bayelva_snow.nc). As UTM was not supported for all steps we anticipated while developing this tutorial (see below), we also created also a lat/lon version of the same data (data/Bayelva_snow_geo.nc)._
